roles/oVirt.datacenter-cleanup/README.md

```diff
@@ -17,6 +27,7 @@ Role Variables
 |--------------------------|-----------------------|--------------------------------------|
 | data_center_name         | UNDEF                 | Name of the data center.             |
 | format_storages          | false                 | Whether role should format storages when removing them. |
+| hosted_engine_vm_name    | HostedEngine          | Name of the hosted engine VM. If VM with such name exists HE environment is assumed. |
```

Comment:
What is the reason of passing the name? Why can't you fetch the `host_engine` VM based on API results. There is `<origin>` element in xml which indicates if VM is hosted engine. Any reason to not use it?

Reply:
Good point. I am reposting on top of Tareqs patch.